client/components/CreateProfileAddInfo.js

```diff
@@ -0,0 +1,189 @@
+
+            onSubmit={ values => {
+
+              const newProfileObject = this.props.navigation.getParam('newProfile', {});
+              const { company, nationality, country,  place, linkedIn, github, cv, website } = values;
+
+                Keyboard.dismiss();
+                let completedProfile = populateNewProfile( newProfileObject, company, nationality, country,  place, linkedIn, github, cv, website);
+                
+                delete completedProfile._id;
+                
+                ( async () => {
+                await this.createMyProfileInDB(completedProfile);
+                console.log('COMPLETED PROFILE', completedProfile);
+                  this.props.navigation.navigate('Dashboard');
+                })();
+              }
+            }>
+          {({ handleChange, handleSubmit, values, handleBlur }) => (
+
+            <ScrollView 
+              contentContainerStyle={styles.wrapper}
+              showsVerticalScrollIndicator={false}
+              >
+            <TextField
+                style={styles.formField}
+                onChangeText={handleChange('company')}
+                onBlur={handleBlur('company')}
+                value={values.company}
+                label="Company"
+                placeholder="Company"
+              />
+
+              <TextField
+                style={styles.formField}
+                onChangeText={handleChange('nationality')}
+                value={values.nationality}
+                label="Nationality"
+                placeholder="Nationality"
+              />
+
+              <TextField
+                style={styles.formField}
+                onChangeText={handleChange('country')}
+                value={values.country}
+                label="Country"
+                placeholder="Country"
+              />
+
+              <TextField
+                style={styles.formField}
+                onChangeText={handleChange('place')}
+                value={values.place}
+                label="Place"
+                placeholder="Place"
+              />
+
+              
+              <TextField
+                style={styles.formField}
+                onChangeText={handleChange('linkedIn')}
+                value={values.linkedIn}
+                label="linkedIn"
+                placeholder="linkedIn"
+              />
+
+              <TextField
+                style={styles.formField}
+                onChangeText={handleChange('github')}
+                value={values.github}
+                label="GitHub"
+                placeholder="GitHub"
+              />
+
+              <TextField
+                style={styles.formField}
+                onChangeText={handleChange('cv')}
+                value={values.cv}
+                label="CV"
+                placeholder="CV"
+              />
+
+              <TextField
```

Comment:
With the previous comment this even works better. You can make `CustomTextField` available through files and reuse it across the app.